cmd/tdex/feeder.go

```diff
@@ -0,0 +1,305 @@
+package main
+
+import (
+	"fmt"
+
+	daemonv2 "github.com/tdex-network/tdex-daemon/api-spec/protobuf/gen/tdex-daemon/v2"
+	tdexv2 "github.com/tdex-network/tdex-daemon/api-spec/protobuf/gen/tdex/v2"
+
+	"github.com/urfave/cli/v2"
+)
+
+var (
+	feeder = cli.Command{
+		Name:  "feeder",
+		Usage: "manage price feeds",
+		Subcommands: []*cli.Command{
+			addPriceFeed, startPriceFeed, stopPriceFeed, updatePriceFeed,
+			removePriceFeed, getPriceFeed, listPriceFeeds, listSources,
+		},
+	}
+	addPriceFeed = &cli.Command{
+		Name:   "add",
+		Usage:  "add a new price feed",
+		Action: addPriceFeedAction,
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:  "base_asset",
+				Usage: "base asset of the market for which the price feed is created",
+			},
+			&cli.StringFlag{
+				Name:  "quote_asset",
+				Usage: "quote asset of the market for which the price feed is created",
+			},
```

Comment:
In other commands, we source the market from the CLI state instead of making use of flags to define market's base/quote asset, like done for example for the [`marketfragmenter split`](https://github.com/tdex-network/tdex-daemon/blob/v1/cmd/tdex/marketfragmenter.go#L143-L146) one.

Let's stick with this approach for the moment, therefore we can drop base/quote asset flags from command.

Also, just to make you aware of this, we are planning to refer to a market with label or univoque id instead of its asset pair. The ticker is yet to be opened though.

Reply:
Done